src/main/java/us/model/Payment.java

```diff
@@ -15,7 +15,52 @@
     })
     private Participation participation;
     private String name;
+    private String ssoId;
```

Comment:
It is temporary.
Because I want `detail_meeting `to give `ssoId `to `payment `in controller
But payment just have name, amount, participation
How can I do that?